main/src/ca/uwaterloo/flix/language/phase/Lowering.scala

```diff
@@ -968,6 +968,7 @@ object Lowering {
           val tagSym = visitRestrictableCaseSymUse(sym)
           val p = LoweredAst.Pattern.Tag(tagSym, pat1, tpe, loc)
           LoweredAst.MatchRule(p, None, e)
+        case TypedAst.RestrictableChoosePattern.Error(_, loc) => throw InternalCompilerException("unexpected error restrictable choose pattern near", loc)
```

Comment:
Is it standard to use "near" in ICEs?

Reply:
I don't think so, especially since ICE doesn't print it location, so it will always just say `unexpected error restrictable choose pattern near`

Reply:
Not really. In some places we do it. We should probably think of something. 

The location is already there, so not super relevant to repeat.

Reply:
the location is not not printed though, so you cant see it

Reply:
throw ICE(msg, loc) and throw ICE(msg, SourceLocation.Unknown) gives the exact same information

Reply:
So what do we agree on?

Reply:
I'll remove `near` for now but we should figure something out. I'll bring this up next meeting, maybe just a quick vote?